File: modul5/C/PRAK504-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>

int reverse(int nilai){
    int nilaireverse = 0;
    int sisa = 0;
    while(nilai != 0){
        sisa = nilai % 10;
        nilaireverse = nilaireverse * 10;
        nilaireverse = nilaireverse + sisa;
        nilai = nilai / 10;
    }
    return nilaireverse;
}
int main(){
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C=A+B;
    printf("%d",reverse(C));
}

File: modul5/Python/PRAK504-2310817210029-Putra_Whyra_Pratama_Setiawan.py
def reverse(nilai):
    nilaireverse = 0
    sisa = 0
    while nilai != 0:
        sisa = nilai % 10
        nilaireverse = nilaireverse * 10
        nilaireverse = nilaireverse + sisa
        nilai = nilai // 10
    return nilaireverse

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))